modules/swagger-codegen/XhhGitIgnore/sdk_unit_testing_binary.json

```diff
@@ -1,67 +0,0 @@
-{
```

Comment:
@wing328 I think this file was added on accident?  Please let me know and I can restore them if needed.